sympy/solvers/solveset.py

```diff
@@ -1487,15 +1487,15 @@ def add_type(lhs, rhs, x):
     adding an equation solver:
 
     - Naming conventions:
-      Name of the *identification* *helper* should be as
-      `\_is\_class` where `class` will be the name or abbreviation
-      of the class of equation. The *solving* *helper* will be named as
-      `\_solve\_class`.
+      Name of the ``identification helper`` should be as
```

Comment:
Why is this rendered as code?

Reply:
Thought was to highlight it to make it different from normal text. Is there any better way to highlight text?

Reply:
Yeah use bold or italics, like it was.

Reply:
I think it okay like this